src/test/resources/org/broadinstitute/hellbender/tools/walkers/annotator/VariantAnnotator/AlleleTrim.vcf

```diff
@@ -0,0 +1,172 @@
591.1:n.2442A>T||||||||||0&1|||||||||,C|ENSG00000158062|ENST00000475591|Transcript|non_coding_transcript_exon_variant&non_coding_transcript_variant|2442|||||rs1134583&COSM1602122|2||-1|UBXN11|HGNC|30600|retained_intron|||||||||13/13|||ENST00000475591.1:n.2442A>G||||||||||0&1|||||||||,A|ENSG00000158062|ENST00000494942|Transcript|non_coding_transcript_exon_variant&non_coding_transcript_variant|2227|||||rs1134583&COSM1602122|1||-1|UBXN11|HGNC|30600|retained_intron|||||||||10/10|||ENST00000494942.1:n.2227A>T||||||||||0&1|||||||||,C|ENSG00000158062|ENST00000494942|Transcript|non_coding_transcript_exon_variant&non_coding_transcript_variant|2227|||||rs1134583&COSM1602122|2||-1|UBXN11|HGNC|30600|retained_intron|||||||||10/10|||ENST00000494942.1:n.2227A>G||||||||||0&1|||||||||,A|ENSG00000158062|ENST00000374221|Transcript|missense_variant|1694|1480|494|S/C|Agt/Tgt|rs1134583&COSM1602122|1||-1|UBXN11|HGNC|30600|protein_coding||CCDS41288.1|ENSP00000363338|UBX11_HUMAN||UPI00004700E1|deleterious(0.01)|unknown(0)|16/16||Low_complexity_(Seg):Seg&PROSITE_profiles:PS50099|ENST00000374221.3:c.1480A>T|ENSP00000363338.3:p.Ser494Cys|||||||||0&1|||||||||,C|ENSG00000158062|ENST00000374221|Transcript|missense_variant|1694|1480|494|S/G|Agt/Ggt|rs1134583&COSM1602122|2||-1|UBXN11|HGNC|30600|protein_coding||CCDS41288.1|ENSP00000363338|UBX11_HUMAN||UPI00004700E1|deleterious(0.03)|unknown(0)|16/16||Low_complexity_(Seg):Seg&PROSITE_profiles:PS50099|ENST00000374221.3:c.1480A>G|ENSP00000363338.3:p.Ser494Gly|||||||||0&1|||||||||,A|ENSG00000158062|ENST00000472155|Transcript|non_coding_transcript_exon_variant&non_coding_transcript_variant|2545|||||rs1134583&COSM1602122|1||-1|UBXN11|HGNC|30600|retained_intron|||||||||14/14|||ENST00000472155.1:n.2545A>T||||||||||0&1|||||||||,C|ENSG00000158062|ENST00000472155|Transcript|non_coding_transcript_exon_variant&non_coding_transcript_variant|2545|||||rs1134583&COSM1602122|2||-1|UBXN11|HGNC|30600|retained_intron|||||||||14/14|||ENST00000472155.1:n.2545A>G||||||||||0&1|||||||||,A|ENSG00000158062|ENST00000374223|Transcript|missense_variant|1212|751|251|S/C|Agt/Tgt|rs1134583&COSM1602122|1||-1|UBXN11|HGNC|30600|protein_coding|||ENSP00000363340|||UPI00004700DA|deleterious(0.03)|unknown(0)|12/12||Low_complexity_(Seg):Seg&PROSITE_profiles:PS50099|ENST00000374223.1:c.751A>T|ENSP00000363340.1:p.Ser251Cys|||||||||0&1|||||||||,C|ENSG00000158062|ENST00000374223|Transcript|missense_variant|1212|751|251|S/G|Agt/Ggt|rs1134583&COSM1602122|2||-1|UBXN11|HGNC|30600|protein_coding|||ENSP00000363340|||UPI00004700DA|tolerated(0.06)|unknown(0)|12/12||Low_complexity_(Seg):Seg&PROSITE_profiles:PS50099|ENST00000374223.1:c.751A>G|ENSP00000363340.1:p.Ser251Gly|||||||||0&1|||||||||,A||ENSR00000533083|RegulatoryFeature|regulatory_region_variant||||||rs1134583&COSM1602122|1||||||regulatory_region||||||||||||||||||||||0&1|||||||||,C||ENSR00000533083|RegulatoryFeature|regulatory_region_variant||||||rs1134583&COSM1602122|2||||||regulatory_region||||||||||||||||||||||0&1|||||||||
```

Comment:
Good lord, this is huge.  Is this funcotator output?  Do you really need all those transcripts to test what you want to test?

Reply:
It was a ported GATK4 test. Im guessing it was some pathological Oncotator output that caused an error in GATK3.